.pep8speaks.yml

```diff
@@ -0,0 +1,40 @@
+# File : .pep8speaks.yml
+
+scanner:
+    diff_only: False  # If False, the entire file touched by the Pull Request is scanned for errors. If True, only the diff is scanned.

```

Comment:
No, we just talked about this Change. For future implementation. 
Anyways i think it is a good change, because we can check implemented code changes directly for pep8 standard.